internal/compiler-bridge/src/main/scala/xsbt/ExtractUsedNames.scala

```diff
@@ -162,6 +162,7 @@ class ExtractUsedNames[GlobalType <: CallbackGlobal](val global: GlobalType) ext
         def usedNameInImportSelector(name: Name): Unit = {
           if (!isEmptyName(name) && (name != nme.WILDCARD) && !names.contains(name)) {
             names += name
+            ()
```

Comment:
TBH in most PRs where `()` is added sombody is asking about it.

@eed3si9n @jvican @dwijnand Maybe we should remove that compiler setting?